packages/landing/src/components/daterange.tsx

```diff
@@ -35,6 +36,8 @@ export class DateRange extends Component {
     Config.map.dateRange.dateAfter = this.state.dateAfter;
     Config.map.dateRange.dateBefore = this.state.dateBefore;
     Config.map.emit('dateRange', this.state);
+    const dateRangeSearch = '?' + MapConfig.toUrl(Config.map);
+    window.history.replaceState(null, '', dateRangeSearch);
```

Comment:
we shouldnt change the window object in here, we can do it from the dateRange event?